transport-native-epoll/src/main/java/io/netty/channel/epoll/FileDescriptor.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel.epoll;
+
+import java.io.IOException;
+
+public final class FileDescriptor {
```

Comment:
Seems like it's generic enough and should not belong to the epoll package? We could probably define a `FileDescriptor` interface at `io.netty.channel` and provide `EpollFileDescriptor`?